src/component-library/theme/theme.base.css

```diff
@@ -39,7 +39,9 @@
   --font-weights-bold: 700;
 
   --colors-neutral-white: #ffffff;
+  --colors-neutral-white-05: rgba(255, 255, 255, 0.05);
   --colors-neutral-white-06: rgba(255, 255, 255, 0.06);
+  --colors-neutral-lighter-grey: #dadada;
```

Comment:
Not sure what to do with this grey @tomjeatt

Reply:
This is fine for now, but we'll need to go through this sooner rather than later and make sure we're handling this consistently.